OpenRA.Game/WVec.cs

```diff
@@ -82,7 +82,7 @@ public static WVec LerpQuadratic(WVec a, WVec b, WAngle pitch, int mul, int div)
 
 			// Add an additional quadratic variation to height
 			// Uses fp to avoid integer overflow
-			var offset = (int)((float)((float)(b - a).Length * pitch.Tan() * mul * (div - mul)) / (float)(1024 * div * div));
```

Comment:
`decimal` math is implemented in software, and doesn't use the FPU (with all the trouble that brings).